app/src/org/commcare/tasks/DataPullTask.java

```diff
@@ -504,7 +506,7 @@ private static void recordSuccessfulSyncTime(String username) {
 
         //Wipe storage
         //TODO: move table instead. Should be straightforward with sandboxed db's

```

Comment:
It is, the better strategy here would basically be to stage an entirely new sandbox and restore into that, it's just much more complex.